packages/core/src/domain/automaticErrorCollection.spec.ts

```diff
@@ -58,7 +58,9 @@ describe('console tracker (console-stack disabled)', () => {
 
   it('should format error instance', () => {
     console.error(new TypeError('hello'))
-    expect((notifyError.calls.mostRecent().args[0] as RawError).message).toContain('console error: TypeError: hello')
+    expect((notifyError.calls.mostRecent().args[0] as RawError).message).toMatch(
+      /^console error: TypeError: hello\s+at/
```

Comment:
FMU, in this case, the stacktrace is only one part of formatting process (like adding console.error, error type, error message), any reason to focus on the stacktrace in the test description?

Reply:
Because it's what makes the difference between error formatting with console-stack flag disabled vs when enabled. Having the same test description "should format error instance" for both cases sounds like they have the same behaviour.

Reply:
yes, it is just that formatting an error instance don't have the same meaning in both cases.
Nb: only one of those tests will survive

Reply:
Makes sense.